src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -229,11 +242,49 @@ export class DetailsViewCommandBar extends React.Component<
 
     private renderLoadAssessmentButton = (): JSX.Element | null => {
         if (this.props.featureFlagStoreData.saveAndLoadAssessment) {
```

Comment:
I think we can use FlaggedComponent here instead of using the feature flag like this.